pkg/model/forward_test.go

```diff
@@ -190,3 +190,37 @@ func TestForward_less(t *testing.T) {
 		})
 	}
 }
+
+func TestForwardExtended_MarshalYAML(t *testing.T) {
+	tests := []struct {
+		name     string
+		expected string
+		data     Forward
+	}{
+		{
+			name:     "service-name",
+			expected: "8080:svc:9090",
+			data:     Forward{Local: 8080, Remote: 9090, Service: true, ServiceName: "svc"},
+		},
+		{
+			name:     "service-name-and-labels",
+			expected: "8080:svc:5214",
```

Comment:
shouldn't the expected value be different since now there are labels involved now?

Reply:
This will happen only when labels are passed as values and no service name is given (if a user adds labels and a service name it will return an error). The forward manager will look for the service name that has those labels and set the service name to the correct name. Finally you want to show the user which service are you exposing not where it comes from.